packages/fury-adapter-oas3-parser/lib/parser/oas/parseComponentsObject.js

```diff
@@ -48,9 +49,10 @@ const parseComponentMember = R.curry((context, parser, member) => {
   // Create a Member Element with `member.key` as the key
   const Member = R.constructN(2, context.namespace.elements.Member)(member.key);
 
-  const parseResult = pipeParseResult(context.namespace,
-    parser(context),
-    Member)(member.value);
+  const parseResult = R.map(
```

Comment:
NOTE, this minor refactoring of this code is required to support referencing a component that has multiple values. For example a "response object" in OAS 3 maps to multiple HttpResponse elements. The map is simpler than previous implementation where we are mapping (wrapping) all non annotation elements in members.